xamarin/Xamarin.Forms
# [UWP] Implement CollectionView Scrolled event

### Description of Change ###

Implement CollectionView Scrolled event.

### Issues Resolved ### 

- fixes #8508

### API Changes ###
 
 None

### Platforms Affected ### 

- UWP

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

#### Before
![issue8508-before](https://user-images.githubusercontent.com/6755973/69052460-5d9f6f00-0a07-11ea-8cd7-958e7b0d861c.gif)
#### After
![issue8508-after](https://user-images.githubusercontent.com/6755973/69052461-5d9f6f00-0a07-11ea-8243-a710b340bf01.gif)


### Testing Procedure ###
Launch Core Gallery and navigate to Issue8508. Scroll in the CollectionView below. If an alert appears, the Scrolled method has been invoked and the test has passed.

### PR Checklist ###
